src/EngineManager.php

```diff
@@ -61,7 +62,7 @@ public function createAlgoliaDriver()
         }
 
         if (is_int($batchSize = config('scout.algolia.batch_size'))) {
-            $config->setBatchSize($batchSize);
+            (new Event())->setBatchSize($batchSize);
```

Comment:
Does this batch size actually persist and get used by the client anywhere?

Reply:
Changed to add `batchSize` directly to `SearchConfig`.